power_grid_model_c/power_grid_model/include/power_grid_model/three_phase_tensor.hpp

```diff
@@ -100,7 +100,6 @@ template <bool sym>
 using RealDiagonalTensor = std::conditional_t<sym, double, three_phase_tensor::DiagonalTensor<double>>;
 template <bool sym>
 using ComplexDiagonalTensor = std::conditional_t<sym, DoubleComplex, three_phase_tensor::DiagonalTensor<DoubleComplex>>;
-
```

Comment:
please re-add this whiteline for readability

Reply:
blame --name "clang-format"

Reply:
Resolved in latest commit.